nose_launchable/plugin.py

```diff
@@ -26,9 +26,6 @@ class Launchable(Plugin):
     def __init__(self):
         super().__init__()
 
-        self._client = LaunchableClientFactory.prepare()
```

Comment:
This is where the problem happens. I decided that the best way to recover gracefully from this is to do this in `configure` where a bunch of other checks are happening